src/Backend/Core/Engine/Meta.php

```diff
@@ -477,6 +477,11 @@ public function save($update = false)
             $custom = null;
         }
 
+        //serialize data for save
+        if (!empty($this->data['data'])) {
+            $this->data['data'] = serialize($this->data['data']);
```

Comment:
Thanks for the info, are there any other 'tricks'? Can i cancel this PR and create a new one from a seperate branch?